doc/classes/ProjectSettings.xml

```diff
@@ -1030,6 +1030,10 @@
 			Size of the hash table used for the broad-phase 2D hash grid algorithm.
 			[b]Note:[/b] Not used if [member ProjectSettings.physics/2d/use_bvh] is enabled.
 		</member>
+		<member name="physics/2d/bvh_collision_margin" type="float" setter="" getter="" default="1.0">
+			Additional expansion applied to object bounds in the 2D physics bounding volume hierarchy. This can reduce BVH processing at the cost of a slightly coarser broadphase, which can stress the physics more in some situations.
+			The default value will work well in most situations. A value of 0.0 will turn this optimization off, and larger values may work better for larger, faster moving objects.
```

Comment:
It's true there are a lot of options now, but I'm not sure how to fix it without breaking compatibility, so maybe we can live with it in 3.x.

In 4.0, there's only the bvh available so it will be a bit better, but it would be worth checking if things can be better organized before we're in beta phase.

I would just add a note similar to what we do for hashgrid options, something like:
`[b]Note:[/b] Used only if [member ProjectSettings.physics/2d/use_bvh] is enabled.`